crates/bevy_ui/src/node_bundles.rs

```diff
@@ -25,6 +25,10 @@ use bevy_transform::prelude::{GlobalTransform, Transform};
 pub struct NodeBundle {
     /// Describes the logical size of the node
     pub node: Node,
+    /// Thickness of the node's border
+    ///
+    /// The component is automatically managed by the UI layout system.
```

Comment:
I'd like to see more docs on what changing this value does (and when it can be changed without getting overridden).

Reply:
I kinda wish we could just do private fields with a builder for all of these internal components, but it's not nice ergonomically.

Reply:
I was thinking about ways to decouple rendering and the ui layout algorithm, so users can just add the geometric components and draw whatever they want with the UI renderer but maybe that's too ambitious for now. I'll just put it back to `pub (crate)`.

Reply:
I'm pretty sure making it `pub(crate)` will break the struct update syntax, which would be a really severe ergonomic loss.

Reply:
Oh I got confused about what we were talking about, I meant the fields of the `ComputedBorderThickness` component, not `NodeBundle` etc.

Reply:
Ah, okay! Yes, I'm fine with that change.